src/Core/src/Core/IMenuElement.cs

```diff
@@ -1,9 +1,16 @@
-﻿using Microsoft.Maui.Graphics;
-
-namespace Microsoft.Maui
+﻿namespace Microsoft.Maui
 {
 	public interface IMenuElement : IElement, IImageSourcePart, IText
 	{
+		/// <summary>
+		/// Represents a shortcut key for a MenuItem.
+		/// </summary>
+#if NETCOREAPP || NETSTANDARD2_1_OR_GREATER
+		IAccelerator? Accelerator => null;
```

Comment:
Should this be an `IList` of Accelerator's? I realize that our current `MenuItem` only supports a single `Accelerator` but it seems like that's a `Controls` misfire and not something we should enforce in core?

Reply:
See our convo at https://github.com/dotnet/maui/pull/14740#discussion_r1221413261

Multiple accelerators, while technically possible on Windows, is not on Mac.

Reply:
What about here, where we implement this interface in controls?

https://github.com/dotnet/maui/pull/14740/files#diff-d50392450e23f5e98dff71b062f06cf63614235e6ab8df38ba6a531eec56c6d3R7-R10

Reply:
IList or IEnumerable? (you said IList earlier)